src/server/scripts/BrokenIsles/EyeOfAzshara/instance_eye_of_azshara.cpp

```diff
@@ -171,6 +171,17 @@ struct instance_eye_of_azshara : public InstanceScript
         }
     }
 
+    uint32 GetData(uint32 data) const override
```

Comment:
compilers will probably complain that not everything is returning something, you probably want return 0 after the switch as well